cosmos/lava.json

```diff
@@ -0,0 +1,51 @@
+{
+  "rpc": "https://lava.tendermintrpc.lava.build",
+  "rest": "https://lava.rest.lava.build",
+  "chainId": "lava-mainnet-1",
+  "chainName": "Lava",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lava/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "LAVA",
+    "coinMinimalDenom": "ulava",
+    "coinDecimals": 6,
+    "coinGeckoId": "lava-network",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lava/ulava.png"
+  },
+  "walletUrl": "https://lavanet.xyz",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/lava",
+  "bip44": {
+    "coinType": 118
+  },
+  "bech32Config": {
+    "bech32PrefixAccAddr": "lava@",
+    "bech32PrefixAccPub": "lava@pub",
+    "bech32PrefixValAddr": "lava@valoper",
+    "bech32PrefixValPub": "lava@valoperpub",
+    "bech32PrefixConsAddr": "lava@valcons",
+    "bech32PrefixConsPub": "lava@valconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "LAVA",
+      "coinMinimalDenom": "ulava",
+      "coinDecimals": 6,
+      "coinGeckoId": "lava-network",
```

Comment:
Is it going to be available at the moment when it's launched? If not, please remove the coinGeckoId lines in `currencies`, `feeCurrencies` and `stakeCurrency` for this PR and add it afterwards.

Reply:
yes, there's already an entry without a price, and it will be reflected on mainnet, we work with coingecko